tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/timeline/timeline_component.css

```diff
@@ -61,10 +61,10 @@ limitations under the License.
 .timeline-slider {
   display: inline-block;
   height: 48px;
+  left: 340px; /* Total width of the two buttons and text div to the left. */
   padding: 0;
   position: absolute;
-  right: 0;
-  width: 440px;
+  width: calc(100% - 420px); /* `left` plus some room on the right. */
```

Comment:
I think there was a miscommunication. I think there should be `left` and `right` and no `width` with calc. If you want to keep it, please document where this magical number of `420` is coming from: (It looks like `left` + 80px of margin)

Reply:
Done.